MyBot/MyBot.cpp

```diff
@@ -18,18 +18,22 @@ int main()
 	/* Output simple log messages to stdout */
 	bot.on_log(dpp::utility::cout_logger());
 
-	/* Handle slash command */
-	bot.on_slashcommand([](const dpp::slashcommand_t& event) {
-		if (event.command.get_command_name() == "ping") {
-			event.reply("Pong!");
-		}
-	});
-
 	/* Register slash command here in on_ready */
 	bot.on_ready([&bot](const dpp::ready_t& event) {
 		/* Wrap command registration in run_once to make sure it doesnt run on every full reconnection */
 		if (dpp::run_once<struct register_bot_commands>()) {
-			bot.global_command_create(dpp::slashcommand("ping", "Ping pong!", bot.me.id));
+			std::vector<dpp::slashcommand> commands {
+				{ "ping", "Ping pong!", bot.me.id }
+			};
+
+			bot.global_bulk_command_create(commands);
+		}
+	});
+
+	/* Handle slash command with the most recent addition to D++ features, coroutines! */
+	bot.on_slashcommand([](const dpp::slashcommand_t& event) -> dpp::task<void> {
+		if (event.command.get_command_name() == "ping") {
+			co_await event.co_reply("Pong!");
 		}
```

Comment:
doesnt this need explicit co_return?

Reply:
Not with a co_await

Reply:
I'll just add it anyways because people will forget when it matters